### PR TITLE
add missing types

### PR DESCRIPTION
**Pull request by auipga**
ts version has been identical to the js version

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 59c90b32676f6db8d1050a3a2916154053a4af72

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 59c90b32676f6db8d1050a3a2916154053a4af72

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/01-building-your-application/03-rendering/03-composition-patterns.mdx [hunk 1]
```diff
@@ -283,7 +283,7 @@ import { createContext } from 'react'
 
 export const ThemeContext = createContext({})
 
-export default function ThemeProvider({ children }) {
+export default function ThemeProvider({ children }: { children: React.ReactNode }) {
```
Comment (leerob) [inline docs/02-app/01-building-your-application/03-rendering/03-composition-patterns.mdx:286]:
```suggestion
export default function ThemeProvider({ children }: {
  children: React.ReactNode
}) {
```

**Comment by delbaoliveira** [comment]
Thank you @auipga 🙏🏼

**Review by delbaoliveira — APPROVED** [review]

**Pull request merged**